client/src/views/clan/ClanView.tsx

```diff
@@ -414,13 +441,26 @@ const ClanView = () => {
         )}
         {/* MEMBERS */}
         <Text bold h4 sx={{ my: 1 }}>{clan.brutes.length}/{clan.limit} {t('brutes')}</Text>
+        <Box sx={{
+          display: 'flex',
+          flexWrap: 'wrap',
+          justifyContent: 'center',
+          marginBottom: '15px',
+          marginTop: '15px'
+        }}
+        >
+          <ButtonGroup aria-label="Basic button group">
+            {Object.values(sortOptions).map((key) => <Button sx={{ color: sort === key ? 'orange' : 'black', backgroundColor: 'secondary.main', }} onClick={() => setSort(key)}>{key}</Button>)}
+          </ButtonGroup>
+        </Box>
+
         <Box sx={{
           display: 'flex',
           flexWrap: 'wrap',
           justifyContent: 'center',
         }}
         >
-          {clan.brutes.map((clanBrute) => (
+          {clan.brutes.sort(sortBrute).map((clanBrute) => (
```

Comment:
Instead of sorting the brutes on every render here, you should sort the brutes in a `useEffect` that listens to any change on the `sort` state:

```ts
useEffect(() => {
	setClan((prevClan) => {
	  const clanBrutes = clan.brutes;

		// Handle the sorting here

		return {
				...prevClan,
				brutes: clanBrutes
		};
	});
}, [sort]);
```